.github/workflows/build-to-review-on-trigger.yml

```
@@ -0,0 +1,68 @@
+name: Build Storybook to Design Review on trigger
+
+on:
+  issue_comment:
+    types: [ created ]
+
+jobs:
+  deploy-for-review:
+    runs-on: ubuntu-latest
+
+    if: |
+      github.event.comment &&
+      github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/review')
+
+    steps:
+      - name: Get PR branch name
+        id: comment-pr-ref
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.issue.owner,
+              repo: context.issue.repo,
+              pull_number: context.issue.number,
+            });
+            return pr.data.head.ref
+
+      - name: Set short branch name
+        run: |
+          echo "BRANCH_NAME=$(echo "${{ steps.comment-pr-ref.outputs.result }}" | grep -oE -- '[A-Z]+-[0-9]+')" >> $GITHUB_ENV
+
+      - name: Checkout PR code based on trigger comment
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ steps.comment-pr-ref.outputs.result }}
+
+      - name: Insert auth font awesome auth token
+        run: echo "@fortawesome:registry=https://npm.fontawesome.com/" > .npmrc && echo "//npm.fontawesome.com/:_authToken=${{ secrets.FONT_AWESOME_AUTH_TOKEN }}" >> .npmrc
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.13'
```

https://github.com/bethinkpl/design-system/blob/b1ba0c55aa4f533c5486e534abb51cd9a8328505/.github/workflows/build.yml#L16
```suggestion
          node-version: '20.9.0'
```